jfrog-cli/jfrog/buildtools_test.go

```diff
@@ -306,23 +306,19 @@ func TestDockerPush(t *testing.T) {
 	if !*tests.TestDocker {
 		t.Skip("Skipping docker test. To run docker test add the '-test.docker=true' option.")
 	}
-	runDockerTest("jfrog_cli_test_image", t)
+	runDockerPushTest("jfrog_cli_test_image", t)
```

Comment:
Since **jfrog_cli_test_image** is used multiple times in this file, maybe it is worth making it a constant.

Reply:
Done